recipes-core/initrdscripts/tegra-flash-init/init-extra.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
If you change this to `/bin/sh`, we shouldn't need to add bash to the RDEPENDS.  I don't think there are any bashisms in this script, so that should be OK.

Reply:
Makes sense. Removed the bash dependency.